src/stores/OverlayStore.ts

```diff
@@ -891,7 +891,7 @@ export class OverlayColorbarSettings {
 
     @computed get tickNum(): number {
         const tickNum = Math.round((this.height / 100.0) * this.tickDensity);
-        return this.height && tickNum > 1 ? tickNum : 1;
+        return this.height && tickNum > 3 ? tickNum : 3;
```

Comment:
please replace constants with something like `MIN_TICK_NUM` for readibility, thanks.

Reply:
Ok. I added `COLORBAR_TICK_NUM_MIN`